File: margin/src/main/java/com/binance4j/margin/dto/MiniOrderInfo.java
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * Trade order information.
 * 
 * @property symbol        Order symbol.
 * @property orderId       Order id.
 * @property clientOrderId Client order id.
 */
@ApiModel("Trade order information.")
data class MiniOrderInfo(
@ApiModelProperty("Order symbol.")
@JsonProperty("symbol") var symbol:String?=null,
@ApiModelProperty("Order id.")
@JsonProperty("orderId") var orderId:Long?=null,
@ApiModelProperty("Client order id.")
@JsonProperty("clientOrderId") var clientOrderId:String?=null)
{
}

File: connectors/src/main/kotlin/com/binance4j/margin/dto/MiniOrderInfo.kt
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * Trade order information.
 * 
 * @property symbol        Order symbol.
 * @property orderId       Order id.
 * @property clientOrderId Client order id.
 */
@ApiModel("Trade order information.")
data class MiniOrderInfo(
@ApiModelProperty("Order symbol.")
 val symbol : String,
@ApiModelProperty("Order id.")
 val orderId : Long,
@ApiModelProperty("Client order id.")
 val clientOrderId : String)
{
}